pkg/kubectl/cmd/attach/attach.go

```diff
@@ -31,6 +31,7 @@ import (
 	"k8s.io/cli-runtime/pkg/genericclioptions/resource"
 	restclient "k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/remotecommand"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/exec"
```

Comment:
Should a command depend on other commands. If we don't this so, can we put a TODO here to fix in the future? Like you said, I don't think we should address this here because the PR is already big.